roles/DCOS.agent_windows/tasks/main.yml

```diff
@@ -0,0 +1,61 @@
+- name: Windows | Defining DC/OS by using 'version' or, more specifically, 'image_commit'
+  set_fact:
+     dcos_version_specifier: "{{ dcos['version'] }}"
+     ansible_password: '{{ hostvars[inventory_hostname].pass }}'
+  no_log: True
+
+
+- name: Windows | Install vcredist140
+  win_chocolatey:
+    name: vcredist140
+    state: present
```

Comment:
it is, but we also need to notify reboot once a new win_sheduled_task added. So for now adding there

Reply:
in fact this one should be , but if we create a new scheduled job, it also needs to happen. INstead of 2 reboots, we ended up with 1 after Scheduled Job creation.